source-google-analytics-data-api/test.flow.yaml

```diff
@@ -0,0 +1,10 @@
+---
+captures:
+  acmeCo/source-google-analytics-v4:
+    shards:
+      logLevel: debug
+    endpoint:
+        connector: 
+          image: "ghcr.io/estuary/source-google-analytics-data-api:b3b4a82"
```

Comment:
So, i got a bit confused on the images names, mainly the ones tagged `DEV`, from what i saw V3 had a `DEV` tag and was not the latest snapshot, i imagined it could mean a non-prod version, so i just went with `b3b4a82`
Now that you mentioned it, i do think `b3b4a82` is just a new form of the V3, since it came out later but on the exact same day

Reply:
In the `estuary/airbyte` repo as well as `estuary/connectors` repo, every time a commit is merged to the `main` branch CI runs and builds + pushes a new image for the connector. It will push it as 3 tags: the `dev` tag, the latest "version" tag which is `v3` in the case of `source-google-analytics-data-api`, and the commit tag, like `b3b4a82`. All 3 of these images are identical. The `dev` tag is always the most up to date, so if you just want the latest version you'd use that tag. The "version" tags have some use if breaking changes are made and we need to manage the migration of pre-existing captures from one version to another. The commit tags are rarely used, but are sometimes helpful when doing advanced troubleshooting.

So for the sake of clarity here, we should use the `v3` tag. That will make it obvious that these initial snapshots were created with the `v3` production image.

Reply:
The base image was modified to V3
Snapshots tests were made on the second commit ( where the initial snapshot was added ) and the commit was rebased with V3.
Thankfully, nothing changed, just commited everything.